front/app/modules/commercial/flag_inappropriate_content/admin/containers/Setting/index.tsx

```diff
@@ -7,6 +7,8 @@ import styled from 'styled-components';
 import { IconTooltip, Toggle } from 'cl2-component-library';
 import { injectIntl, FormattedMessage } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
+import { trackEventByName } from 'utils/__mocks__/analytics';
```

Comment:
I think the import here should be from `utils/analytics`.

Reply:
And that's why we do code reviews! Auto-import :) 

Thanks! 🙌